code/modules/antagonists/eldritch_cult/eldritch_antag.dm

```diff
@@ -222,7 +223,7 @@
 	var/timer = 5 MINUTES
 
 /datum/objective/stalk/process()
-	if(owner?.current.stat != DEAD && target?.current.stat != DEAD && (target.current in view(5,owner.current)))
+	if(owner?.current.stat != DEAD && target && target.current.stat != DEAD && (target.current in view(5,owner.current)))
```

Comment:
Why?

Reply:
Check would runtime if there was no target in the distance check so why run that if there is no target

Reply:
Ah, right